docs/src/modules/utils/getProductInfoFromUrl.ts

```diff
@@ -0,0 +1,43 @@
+import { pathnameToLanguage } from 'docs/src/modules/utils/helpers';
+
+export type MuiProductId =
+  | null
+  | 'base-ui'
+  | 'material-ui'
+  | 'joy-ui'
+  | 'system'
+  | 'data-grid'
+  | 'date-pickers';
```

Comment:
I'm unsure about this one. There is a long tail of MUI X components that will be strange to handle as standalone components, like file upload or tree view.

Reply:
I think it would make sense for 
- data grid because the number of options is gigantic
- charts because it will contain multiple components

For components such as tree view, file upload, ... It could be under eXplore naming

Pickers is an in-between. It's small components but with a lot of configuration